### PR TITLE
Nailgun ssl cert verification

### PR DESCRIPTION
**Pull request by dosas**
This PR adds the possibility to overwrite the `verify` settings of nailgun's [ServerConfig ](https://github.com/SatelliteQE/nailgun/blob/master/nailgun/config.py#L219) 

According to the setttings given by [requests](https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification)

Backwards compatibility is kept by defaulting to `False` if the new settings option is not available.

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
test-robottelo

**Comment by omkarkhatavkar** [comment]
"trigger": "test-robottelo"

**Review by jyejare — APPROVED** [review]

File: robottelo/config/__init__.py [hunk 1]
```diff
@@ -99,6 +99,18 @@ def get_url():
...
 
 
+def get_ssl_cert_verify():
+    """Return the SSL certificate or setting to verify HTTPS requests.
+
+    :return: Certificate path or boolean verify setting
+    :rtype: bool or str
+    """
+    try:
+        return settings.server.verify_ca
```
Comment (jyejare) [inline robottelo/config/__init__.py:109]: You can validate settings from https://github.com/SatelliteQE/robottelo/blob/master/robottelo/config/validators.py to return the `False` value by default and hence need not to handle exception here explicitly!

Comment (jyejare) [reply robottelo/config/__init__.py:109]: Also, you don't need this function in that case, just directly call the settings object wherever applicable

**Review by jyejare — COMMENTED** [review]
NonBlocking comments!

**Comment by jyejare** [comment]
trigger: test-robottelo

File: robottelo/config/__init__.py (hunk 1 above)
Comment (dosas) [reply robottelo/config/__init__.py:109]: Thanks, in that case I will adapt it

**Review by omkarkhatavkar — APPROVED** [review]
ACK

**Comment by omkarkhatavkar** [comment]
@jyejare We need to add the same in the CI PR to make this work, otherwise regression would have impact

**Comment by jyejare** [comment]
@omkarkhatavkar Not required, The validator should handle it and return False value automatically! We can merge it as is but good keep CI in sync !

**Comment by jyejare** [comment]
@omkarkhatavkar Opened CI MR 1134 !

**Review by jyejare — APPROVED** [review]

**Pull request merged**